data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -110,7 +110,7 @@ then
 		/usr/bin/cluster-kube-apiserver-operator render \
 		--manifest-etcd-serving-ca=etcd-ca-bundle.crt \
 		--manifest-etcd-server-urls={{.EtcdCluster}} \
-		--manifest-image=${OPENSHIFT_HYPERSHIFT_IMAGE} \
+		--manifest-image=${OPENSHIFT_HYPERKUBE_IMAGE} \
```

Comment:
Took a while to figure out this ... the hyperkube image is what we need since openshift/installer#1885 is part of this rebase, but the resolving conflicts during the rebase had accidentally revert that change. This commit restores it.